phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexScrutinyMapper.java

```diff
@@ -288,8 +301,15 @@ protected void checkIfInvalidRowsExpired(Context context,
             Pair<Long, List<Object>> sourceValues = entry.getValue();
             Long sourceTS = sourceValues.getFirst();
             if (hasRowExpiredOnSource(sourceTS, ttl)) {
-                context.getCounter(PhoenixScrutinyJobCounters.EXPIRED_ROW_COUNT).increment(1);
-                itr.remove();
+                context.getCounter(PhoenixScrutinyJobCounters.EXPIRED_ROW_COUNT).increment(1L);
+                itr.remove(); //don't output to the scrutiny table
+            } else if (isRowOlderThanMaxLookback(sourceTS)){
+                context.getCounter(PhoenixScrutinyJobCounters.BEYOND_MAX_LOOKBACK).increment(1L);
+                //still output to the scrutiny table just in case it's useful
```

Comment:
why do we need them around?

Reply:
Humans need to evaluate beyond max lookback counters to see if they're valid issues or not.

Reply:
@gjacoby126 Any reason to be concerned about bloating the scrutiny table? This is an expected race condition that can happen pretty regularly if rows are updated at the cadence of around MaxLookbackAge. Humans may not necessarily have bandwidth to look at potentially large number of rows. If we are going to emit to scrutiny table, perhaps we should add a way to filter these out as well to make them manageable.